server/tests/src/test/resources/configuration/security/ldap.xml

```diff
@@ -3,13 +3,12 @@
           xmlns="urn:infinispan:server:14.0">
    <security-realms>
       <security-realm name="default">
-         <ldap-realm name="ldap" url="${org.infinispan.test.ldap.url}"
-                     principal="${org.infinispan.test.ldap.principal}" credential="strongPassword">
-            <identity-mapping rdn-identifier="uid" search-dn="${org.infinispan.test.ldap.search-dn}" search-recursive="true">
+         <ldap-realm name="ldap" url="ldap://${org.infinispan.test.host.address}:10389"
```

Comment:
Does this mean we can no longer run tests with external LDAP servers? Did we ever do that?

Reply:
We can run the test with an external LDAP server doing the following: `mvn verify -Dit.test=AuthorizationLDAPIT,AuthenticationLDAPIT -pl server/tests -Dinfinispan.test.checkThreadLeaks=false -Dorg.infinispan.test.ldapServer=remote -Dorg.infinispan.test.ldap.url=ldap://myhost.com -Dorg.infinispan.test.ldap.principal="user" -Dorg.infinispan.test.ldap.credential=pwd -Dorg.infinispan.test.ldap.search-dn="ou=ispn" -Dorg.infinispan.test.hotrod.client.salsmechanism=PLAIN`

Instead of having a lot of properties. For testing purposes replace server/tests/src/test/resources/configuration/security/ldap.xml

Reply:
Are you sure `-Dorg.infinispan.test.ldap.url=ldap://myhost.com` still works? You replaced `url="${org.infinispan.test.ldap.url}"` here with `ldap://${org.infinispan.test.host.address}:10389`

Reply:
> Are you sure -Dorg.infinispan.test.ldap.url=ldap://myhost.com still works? You replaced url="${org.infinispan.test.ldap.url}" here with ldap://${org.infinispan.test.host.address}:10389

Yes. It is working. The file `ldap.xml` has changed because I rollbacked my changes to the original file content. Having a unique `ldap.xml` and `infinispan.ldif` was becoming a nightmare because each LDAP has a different syntax and configuration. Also, I should be adding a lot of properties in order to achieve those replacements that become hard to understand.

The files `ldap.xml` and `infinispan.ldif` must be replaced if we wish to test against a remote LDAP server.

* By default, ISPN will run the test against an Embedded LDAP Server that have all properties filled.
* The required properties to test against a Remote LDAP Server are the following and are used only in `RemoteLdapServer` in order to insert the ldif into the remote LDAP Server
```
public static final String TEST_LDAP_URL = "org.infinispan.test.ldap.url";
public static final String TEST_LDAP_PRINCIPAL = "org.infinispan.test.ldap.principal";
ublic static final String TEST_LDAP_CREDENTIAL = "org.infinispan.test.ldap.credential";
```

Reply:
Ok. I'm not going to run the tests with an external LDAP server any time soon, I just ran the command in your comment and noticed it failed with a parsing error instead of the network error I was expecting.